controllers/operandconfig.go

```diff
@@ -0,0 +1,469 @@
+//
```

Comment:
Why this new file is created?

Reply:
Yeah, I think it is introduced when I rebased some commits.
Updated~